src/Code.gs

```diff
@@ -168,13 +169,7 @@ const ChatGPTApp = (function () {
     .setName("getImageDescription")
     .setDescription("To retrieve the description of an image.")
     .addParameter("imageUrl", "string", "The URL of the image.")
-    .addParameter("fidelity", `"low" | "high"`, `default "low" (sufficient in most cases)`, isOptional = true);
-
-  let getDataFromGoogleSheetsFunction = new FunctionObject()
-    .setName("getDataFromGoogleSheets")
-    .setDescription("To retrieve all the data contained in a Spreadsheet or a sheet.")
-    .addParameter("spreadsheetId", "string", "The ID of the spreadsheet")
-    .addParameter("sheetName", "string", `The name of a specific sheet to access.`, isOptional = true);
+    .addParameter("highFidelity", "boolean", `To improve the image quality, not needed in most cases.`, isOptional = true);
```

Comment:
If this parameter is optional, the default value should be explicitly stated (default false).